app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -271,50 +271,3 @@ $large-input-size: 50px;
     padding-left: 0;
   }
 }
-
-.search-toggle {
-  display: none;
-  background-color: govuk-colour("blue");
-  background-image: url("govuk_publishing_components/search-button.png");
```

Comment:
Can the `search-button.png` image file also be removed?

Reply:
Yep, I agree. That doesn't seem to be used anywhere. Removed now.